src/main/java/cst438flights/controller/flightRestController.java

```diff
@@ -2,18 +2,20 @@
 
 import cst438flights.domain.Flight;
 import cst438flights.domain.FlightInfo;
+import cst438flights.domain.Reservation;
 import cst438flights.service.FlightService;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.PathVariable;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
 
 import java.util.List;
 
 @RestController
 public class flightRestController {
+    //for holding the origin name
+    static final String ORIGIN = "planner";
```

Comment:
Adjusted ORIGIN to equal "planner", now matches reservationOrigin term used in other services.